public/app/percona/tour/steps/product/product.utils.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { TourStep } from 'app/percona/shared/core/reducers/tour';
+import { ServiceType } from 'app/percona/shared/services/services/Services.types';
+
+import SidebarStep from '../../components/SidebarStep';
+
+import { Messages } from './product.messages';
+
+export const getPMMDashboardsStep = (services: ServiceType[]): TourStep => {
```

Comment:
What happens if I have, say, MySQL and PostgreSQL? Will this only return for MySQL?

Reply:
This can be changed to something like:

```
let ariaLabel = 'Operating System (OS)';
let navMenuId = 'system';

for (let i = 0; i < services.length; i++) {
    switch(services[i]) {
        case ServiceType.mysql:
            ariaLabel = 'MySQL';
            navMenuId = 'mysql';
            return; // or break, not sure depending on what we want to achieve
         // ...
    }
}

return getStep(ariaLabel, navMenuId);
```

Reply:
Also, I think we want to show some services preferably if possible (if MySQL, Mongo and ProxySQL are available, we would want to show the MySQL one). So the order is important here and iterating over the services would not fit here.

Reply:
Ok, I think I understand now. Thanks 👍